ASF-STM.user.js

```diff
@@ -912,7 +912,7 @@
             debugPrint(new Date(Date.now()));
             hideThrobber();
             hideMessage();
-            updateProgress(bots.Result.length - 1);
+            updateProgress(bots.Result.length - 1, bots.Result.length);
```

Comment:
you changed formula in `updateProgress` but did not bother to change the `-1` here.